salt/modules/lxd.py

```diff
@@ -3213,8 +3213,44 @@ def image_alias_delete(image,
 #####################
 
 
-def snapshots_all(container=None, remote_addr=None,
-                  cert=None, key=None, verify_cert=True):
+def snapshots_all(container, remote_addr=None, cert=None, key=None, verify_cert=True):
+    '''
+    Get all snapshots for a container
+
+    container :
+        The name of the container to get.
+
+    remote_addr :
+        An URL to a remote Server, you also have to give cert and key if
+        you provide remote_addr and its a TCP Address!
```

Comment:
`An URL to a remote server. The 'cert' and 'key' fields must also be provided if 'remote_addr' is defined.`